File: Coding Exercises/TypeScript/Day28/swapNumbers.ts
function swapNumbers(a: number, b: number): number[] {
  a = a + b;
  b = a - b;
  a = a - b;
  const result: number[] = [a, b];
  return result;
}

const x: number = 5;
const y: number = 10;

const swapped: number[] = swapNumbers(x, y);

console.log(`Valores trocados: x = ${swapped[0]}, y = ${swapped[1]}`);

File: Coding Exercises/Python/Day28/swap_numbers.py
def swap_numbers(a, b):
    a = a + b
    b = a - b
    a = a - b
    result = [a, b]
    return result


x = 5
y = 10

swapped = swap_numbers(x, y)

print(f"Valores trocados: x = {swapped[0]}, y = {swapped[1]}")
